src/java.base/share/classes/jdk/internal/classfile/impl/verifier/VerificationBytecodes.java

```diff
@@ -88,38 +87,9 @@ static boolean is_store_into_local(int code) {
     }
 
     static final int _lengths[] = new int[number_of_codes];
-
-    static int special_length_at(int code, byte bytecode[], int bci, int end) {
-        switch (code) {
-            case WIDE:
-                if (bci + 1 >= end) {
-                    return -1;
-                }
-                return wide_length_for(bytecode[bci + 1] & 0xff);
-            case TABLESWITCH:
-                int aligned_bci = align(bci + 1);
-                if (aligned_bci + 3 * 4 >= end) {
-                    return -1;
-                }
-                ByteBuffer bb = ByteBuffer.wrap(bytecode, aligned_bci + 1 * 4, 2 * 4);
-                int lo = bb.getInt();
-                int hi = bb.getInt();
-                int len = aligned_bci - bci + (3 + hi - lo + 1) * 4;
-                return len > 0 ? len : -1;
-            case LOOKUPSWITCH:
-            case _fast_binaryswitch:
-            case _fast_linearswitch:
-                aligned_bci = align(bci + 1);
-                if (aligned_bci + 2 * 4 >= end) {
-                    return -1;
-                }
-                int npairs = ByteBuffer.wrap(bytecode, aligned_bci + 4, 4).getInt();
-                len = aligned_bci - bci + (2 + 2 * npairs) * 4;
-                return len > 0 ? len : -1;
-            default:
-                return 0;
-        }
-    }
+            case ClassFile.WIDE:
+            case ClassFile.TABLESWITCH:
+            case ClassFile.LOOKUPSWITCH:
```

Comment:
This method was removed in [JDK‑8339576] ([GH‑20863]):
```suggestion
```

[JDK‑8339576]: https://bugs.openjdk.org/browse/JDK-8339576
[GH‑20863]: https://github.com/openjdk/jdk/pull/20863

Reply:
hmm, seems i forgot to push the local fix.